app/operations/event_procedures/build_total_amount_cents.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module EventProcedures
+  class BuildTotalAmountCents < Actor
+    input :attributes, type: Hash
+
+    output :total_amount_cents, type: Integer
+
+    APARTMENT_PERCENTAGE = 1
+    URGENCY_PERCENTAGE = 0.3
+
+    def call
+      urgency, room_type = attributes.values_at(:urgency, :room_type)
+      procedure_amount_cents = find_procedure_amount_cents
+
+      total = procedure_amount_cents
+      total += (procedure_amount_cents * APARTMENT_PERCENTAGE) if room_type == "apartment"
```

Comment:
I feel like we should be doing something like this for this line and for the line below
```suggestion
      total += apartment_amound(procedure_amount_cents) if entity.apartment?
```